pom.xml

```diff
@@ -1502,6 +1502,70 @@
             </properties>
         </profile>
 
+        <profile>
+            <!-- Use the latest version of the standard channels when provisioning. -->
+            <id>latest.standard.channels.profile</id>
+            <activation><property><name>latest.standard.channels</name></property></activation>
+            <properties>
+                <!-- Allow online provisioning so wildfly-maven-plugin can resolve the latest manifests -->
+                <galleon.offline>false</galleon.offline>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <version>${version.org.wildfly.plugin}</version>
+                        <configuration>
+                            <channels>
+                                <channel>
+                                    <manifest>
+                                        <groupId>${channels.maven.groupId}</groupId>
+                                        <artifactId>wildfly-ee</artifactId>
+                                    </manifest>
+                                </channel>
+                                <channel>
+                                    <manifest>
+                                        <groupId>${channels.maven.groupId}</groupId>
+                                        <artifactId>wildfly</artifactId>
+                                    </manifest>
+                                </channel>
+                            </channels>
+                        </configuration>
```

Comment:
There is no wildfly-preview channel attached here, so I'm not sure how the expected channels would be injected when this plugin configuration is merged with the one on preview/build.

Would it become an issue?

I guess the same is applied to the ee-build. The plugin configuration merged will attach wildfly-ee and wildfly channels. Is that expected for an ee-build?

Reply:
In both cases, yes the limitations you point out are expected. This profile as is will not work for testing WFP or for testing wildfly-ee by itself.

https://issues.redhat.com/browse/WFLY-19268 will replace this, but it is out of scope for this task, which is really just about having an upstream variant of an **urgent** downstream change that tests a combination of these two FPs. Neither wildfly-ee-only testing nor wildfly-preview are relevant to the downstream change.

https://issues.redhat.com/browse/WFLY-19268 will take advantage of the greater flexibility we have in wildfly-maven-plugin 5, which is not used downstream.